services/net/image/ImageAction.cs

```diff
@@ -77,7 +77,7 @@ public ImageAction(IKafkaMessenger producer, IApiService api, IOptions<ImageOpti
         var mountPath = String.IsNullOrEmpty(manager.DataSource.GetConnectionValue("inputpath")) ? this.Options.InputPath : GetInputPath(manager.DataSource);
         var inputFileCode = String.IsNullOrEmpty(manager.DataSource.GetConnectionValue("inputfilecode")) ? manager.DataSource.Code : manager.DataSource.GetConnectionValue("inputfilecode");
         var keyFilePath = Path.Combine(this.Options.PrivateKeysPath, filename);
-        if (File.Exists(keyFilePath))
+        if (File.Exists(keyFilePath) || (File.Exists(filename)))
```

Comment:
Shouldn't this be a `&&`?  Also doesn't the `keyFilePath` include the `filename`?  I'm just wondering if `File.Exists(filename)` would ever be true?